Automattic/jetpack
# Remove border CSS from button block

#### Changes proposed in this Pull Request:


👋 👋 

https://github.com/Automattic/jetpack/pull/19595 ensures that the CSS for the core button block is enqueued when a button is rendered on the server. That way, the Jetpack button uses core's button styles. 

However there is still some "default" button CSS that overrides the core + theme styles:

<img width="1904" alt="Screen Shot 2022-03-31 at 12 58 08 PM" src="https://user-images.githubusercontent.com/5375500/161110120-c01e331c-32c4-4304-88d9-1204295dd560.png">

This PR removes that CSS, allowing themes to provide a default value for button borders, fonts, and padding via theme.json that applies to Jetpack buttons.

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






* Sandbox remotedemo.wordpress.com
* Vist /about and view the jetpack contact form button
* Verify the core + theme button styles are not being overridden by Jetpack's
